### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Deeksha2501**
These are some grammar checks I have done in README file
I hope after this no grammar issue will be left in this file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -333,17 +333,17 @@ If you are worried or don't know where to start, you can **always** reach out to
 
 We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributor's Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help.
 
-_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!!
+_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to [webpack@opencollective.com](https://opencollective.com/) and we can give pointers or tips!!!
```
Comment (sokra) [inline README.md:336]: Link is wrong. It should be a email address.

Comment (Deeksha2501) [reply README.md:336]: Ok I will make the changes

Comment (Deeksha2501) [reply README.md:336]: Which email address I have to put here?
When I tried to put "webpack@opencollective.com" it is showing "404 page not found"

**Comment by Deeksha2501** [comment]
I have done the requested changes. Kindly review them

File: README.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ and incredibly **fast** on incremental compilations.
 
 ### Module Formats
 
-webpack supports ES2015+, CommonJS and AMD modules **out of the box**. It performs clever static
```
Comment (Deeksha2501) [inline README.md:297]: "comma" is required

File: README.md [hunk 3]
```diff
@@ -333,17 +333,17 @@ If you are worried or don't know where to start, you can **always** reach out to
 
 We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributor's Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help.
 
-_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!!
```
Comment (Deeksha2501) [inline README.md:336]: changed the link. It must point to the mail address of webpack

File: README.md [hunk 4]
```diff
@@ -407,7 +407,7 @@ If you have discovered a 🐜 or have a feature suggestion, feel free to create
 
 <h2 align="center">Sponsoring</h2>
 
-Most of the core team members, webpack contributors and contributors in the ecosystem do this open source work in their free time. If you use webpack for a serious task, and you'd like us to invest more time on it, please donate. This project increases your income/productivity too. It makes development and applications faster and it reduces the required bandwidth.
+Most of the core team members, webpack contributors, and contributors in the ecosystem do this open-source work in their free time. If you use webpack for a serious task, and you'd like us to invest more time on it, please donate. This project increases your income/productivity too. It makes development and applications faster and it reduces the required bandwidth.
```
Comment (reinaldosimoes) [inline README.md:410]: You invest something "in it" and not "on it"

**Pull request closed without merging**